src/lib/providers/BidProvider.tsx

```diff
@@ -113,7 +137,13 @@ const BidProvider: React.FC<BidProviderType> = ({ children }) => {
 						bidderAccount: id,
 						amount,
 						date: new Date(), // not supported by zAuction
-						tokenId: e.tokenId,
+						tokenId: Number(e.tokenId),
```

Comment:
Bonus points: Consider making a helper function so you don't repeat the same code here that you do in the `getBidsForDomain` this would follow the D.R.Y principle 

The helper could take either DTO's from the endpoints, and return a `Bid`